content/blog/elao/afup-day-2-2021.md

```diff
@@ -0,0 +1,55 @@
+---
+type:               "post"
+title:              "AFUP Day 2021, jour 2 : le compte-rendu de l'équipe"
+date:               "2021-06-11"
+lastModified:       ~
+
+description:        "Notre compte-rendu du deuxième jour de l'Afup Day, édition 2021, à distance !"
+thumbnail:          "images/posts/2021/afup-day-2021/afup-day-2021.png"
+tags:               ["Développement", "afup", "Conférence", "afupDay"]
+categories:         ["conference"]
+author:             ["bleveque"]
+---
+
+Le deuxième jour de conférence de l'AFUP Day a eu lieu le 11 juin 2021, entièrement en ligne. Nous avons pu ainsi suivre les conférences organisées par les antennes locales de Toulouse et Tours. Bravo à eux et à toute l'équipe de l'AFUP pour cette édition très bien organisée qui, malgré la distance, parvient à donner le sourire :).  
+Voici quelques notes de l'équipe d'Elao qui vous propose de faire un zoom sur trois conférences :
+
+## Comment dompter un Legacy ? Parlons smoke testing, golden master et même migration progressive, par Estelle Le Cam
+
+Retour d'expérience sur la migration continue d'un projet initialement développé en PHP 4 et qui a subi plusieurs migrations successives (PHP 5 avec un framework maison, puis PHP7 avec un framework opensource).
```

Comment:
Pourquoi ne pas précisé que le framework opensource est Symfony?

Reply:
parce que je n'avais pas noté que c'était Symfony